packages/migrate/src/__tests__/MigrateStatus.test.ts

```diff
@@ -35,66 +35,61 @@ describe('sqlite', () => {
-
-            P1003: Database dev.db does not exist at dev.db
-          `)
+    await expect(result).rejects.toMatchInlineSnapshot(`P1003: Database dev.db does not exist at dev.db`)
 
     expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
       Prisma schema loaded from prisma/empty.prisma
       Datasource "my_db": SQLite database "dev.db" at "file:dev.db"
-
     `)
-    expect(ctx.mocked['console.log'].mock.calls).toMatchSnapshot()
-    expect(ctx.mocked['console.error'].mock.calls).toMatchSnapshot()
+    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchSnapshot()
+    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchSnapshot()
   })
 
   it('existing-db-1-failed-migration', async () => {
     ctx.fixture('existing-db-1-failed-migration')
+    const mockExit = jest.spyOn(process, 'exit').mockImplementation((number) => {
+      throw new Error('process.exit: ' + number)
+    })
 
     const result = MigrateStatus.new().parse([])
-    await expect(result).resolves.toMatchInlineSnapshot(`
-            The failed migration(s) can be marked as rolled back or applied:
-                  
-            - If you rolled back the migration(s) manually:
-            prisma migrate resolve --rolled-back "20201231000000_failed"
-
-            - If you fixed the database manually (hotfix):
-            prisma migrate resolve --applied "20201231000000_failed"
-
-            Read more about how to resolve migration issues in a production database:
-            https://pris.ly/d/migrate-resolve
-          `)
+    await expect(result).rejects.toMatchInlineSnapshot(`process.exit: 1`)
 
     expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
       Prisma schema loaded from prisma/schema.prisma
       Datasource "my_db": SQLite database "dev.db" at "file:dev.db"
       1 migration found in prisma/migrations
 
+    `)
+    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchSnapshot()
+    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`
       Following migration have failed:
       20201231000000_failed
 
       During development if the failed migration(s) have not been deployed to a production database you can then fix the migration(s) and run prisma migrate dev.
 
+      The failed migration(s) can be marked as rolled back or applied:
+            
+      - If you rolled back the migration(s) manually:
+      prisma migrate resolve --rolled-back "20201231000000_failed"
+
+      - If you fixed the database manually (hotfix):
+      prisma migrate resolve --applied "20201231000000_failed"
+
+      Read more about how to resolve migration issues in a production database:
+      https://pris.ly/d/migrate-resolve
     `)
-    expect(ctx.mocked['console.log'].mock.calls).toMatchSnapshot()
-    expect(ctx.mocked['console.error'].mock.calls).toMatchSnapshot()
+    expect(mockExit).toHaveBeenCalledWith(1)
+    mockExit.mockRestore()
```

Comment:
When a failed migration was found, it was returning a successful error code (0), it now returns 1